tests/02_eradiate/01_unit/scenes/atmosphere/test_molecular_atmosphere.py

```diff
@@ -39,27 +39,20 @@ def test_molecular_atmosphere_afgl_1986(mode_ckd):
         mi_params.update(params.render(ctx))
 
 
-@pytest.fixture
-def ussa76_approx_test_absorption_data_set():
-    return data_store.fetch("tests/spectra/absorption/us76_u86_4-spectra-4000_25711.nc")
-
-
-def test_molecular_atmosphere_ussa_1976(
-    mode_mono, ussa76_approx_test_absorption_data_set
-):
+@pytest.mark.slow
+def test_molecular_atmosphere_ussa_1976(mode_mono):
     # ussa_1976() constructor produces a valid kernel dictionary
+    eval_w = [440.0, 550.0, 660.0] * ureg.nm
     atmosphere = MolecularAtmosphere.ussa_1976(
-        absorption_dataset=ussa76_approx_test_absorption_data_set,
+        wavelength_range=np.stack([eval_w[0], eval_w[-1]])
```

Comment:
This is odd, why not a more explicit `wavelength_range=[eval_w.m[0], eval_w.m[-1]] * eval_w.u)`?

Reply:
How is that more explicit?

Reply:
Two reasons: (i) Your implementation hides fancy unit manipulation done by Pint's internals and (ii) using `np.stack()` to construct an array from scalar value is not a very common workflow. You also construct 3 Pint objects in the process, vs 1 in the solution I'm suggesting.

Reply:
I'll do as you prefer